api_io_sign.yaml

```diff
@@ -137,6 +211,226 @@ definitions:
       - terms_and_conditions_url
       - nonce
 
+  Id:
```

Comment:
cant we refer to these types from the remote io-sign specs?

I mean using  `$ref: "https://raw.githubusercontent.com/io-sign/....`

Reply:
Do you think this could also be true for other entities? I have some doubts about the different openapi version (2.0 vs 3.0)

Reply:
> Do you think this could also be true for other entities

yes

> I have some doubts about the different openapi version (2.0 vs 3.0)

Why do we need a different version here?

Reply:
> Why do we need a different version here?

I saw that all the open APIs of the other services are in 2.0. I think for a compatibility issue

Reply:
It looks like io-app is using the latest codegen version, so we can safely use openapi 3.x cc @hevelius